### DISCUSSION TITLE
Data Provider Developer Experience Feedback

### DISCUSSION THREAD

**Discussion by butchtm** [thinkingmachines/unicef-ai4d-research-bank#122] [General]
Hi @AnthonyMockler 

We're looking to see what we could improve with our[ contribution guidelines documentation](https://github.com/thinkingmachines/unicef-ai4d-research-bank/blob/main/catalog-contribution.md) and we are actively seeking feedback on an external data providers' developer experience when they try to add a new catalog item (either a dataset/model) to the research bank. 

Maybe you can open this up to your colleagues and other future contributors?

**Comment by AnthonyMockler** [comment]
Done @butchtm ! I've asked my colleague Hugo to have a go at this

**Comment by butchtm** [comment]
Adding a [reference](https://github.com/thinkingmachines/unicef-ai4d-research-bank/issues/3#issuecomment-1550967233) to @huruizverastegui feedback
